go_expvar/datadog_checks/go_expvar/go_expvar.py

```diff
@@ -170,12 +170,17 @@ def parse_expvar_data(self, data, tags, metrics, max_metrics, namespace):
                 try:
                     float(value)
                 except (TypeError, ValueError):
-                    if isinstance(value, dict):
-                        self.log.warning(
-                            "Path %s is a mapping; specify a more specific path to report a value: %s",
-                            actual_path,
-                            value,
-                        )
+                    if isinstance(value, dict) or isinstance(value, list):
+                        if actual_path == DEFAULT_MAPPING:
+                            self.log.debug(
+                                "Skipping configured path %s; most memstats metrics are collected by default."
+                                "Specify a more specific path to report a new value",
+                                actual_path,
+                            )
+                        else:
+                            self.log.warning(
+                                "Path %s is a mapping; specify a more specific path to report a value", actual_path
```

Comment:
Should this log line be updated? 
```suggestion
                                "Path cannot be a mapping or array ; specify a more specific path to report a value", actual_path
```